### DISCUSSION TITLE
ADS-B data

### DISCUSSION THREAD

**Discussion by 7omoud** [Oros42/IMSI-catcher#149] [General]
I like to get the airplane informations on my harckRF portapack! Unfortunately when I found airplane I can't get the information of the plan because the device said "No airlines.do found!

Where can I download the file? 
![image](https://user-images.githubusercontent.com/106143874/170720195-b9cfcc11-01c2-43ea-bdec-8e2177df8476.jpeg)

**Comment by Oros42** [comment]
Wrong place for this.  
My program is not design for ADS-B.
